diagnostic.md

```diff
@@ -3,67 +3,72 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+element.style
+div
+body
+html
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$( "OL" ).children().css();
+$( "OL" ).find( "li" ).css();
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$( ".listerItem" ).parents("form").css();
 
 ## Question 4
 
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+getter
+find()
+text()
 
-<!-- your answer ends here -->
+setter
+append()
+prepend()
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is the browser's object-ized representation of a page.
```

Comment:
Yes and it is modeled like a tree so that everything is either a child or parent of another node, most times they are both parents and children.